mifospay/src/main/java/org/mifos/mobilewallet/mifospay/settings/ui/SettingsScreen.kt

```diff
@@ -0,0 +1,129 @@
+package org.mifos.mobilewallet.mifospay.settings.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.designsystem.component.MifosItemCard
+import org.mifos.mobilewallet.mifospay.designsystem.component.MifosTopBar
+import org.mifos.mobilewallet.mifospay.designsystem.theme.mifosText
+import org.mifos.mobilewallet.mifospay.designsystem.theme.styleSettingsButton
+import org.mifos.mobilewallet.mifospay.ui.utility.DialogState
+import org.mifos.mobilewallet.mifospay.ui.utility.DialogType
+
+/**
+ * @author pratyush
+ * @since 12/02/2024
+ */
+
+@Composable
+fun SettingsScreen(
+    backPress: () -> Unit,
+    disable: () -> Unit,
+    logout: () -> Unit
+) {
+    var dialogState by remember { mutableStateOf(DialogState()) }
+
+    DialogManager(
+        dialogState = dialogState,
+        onDismiss = { dialogState = DialogState(type = DialogType.NONE) }
+    )
```

Comment:
I have made a generic DialogManager.kt composable using which we can access both the disable and logout button. Please review this @therajanmaurya

Reply:
It's great!